dapp/src/components/Footer.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+import {
+  Box,
+  useMultiStyleConfig,
+  List,
+  ListItem,
+  LinkProps,
+  Link,
+  Button,
+  Icon,
+} from "@chakra-ui/react"
+import { EXTERNAL_HREF } from "#/constants"
+import { AcreSignIcon, ArrowUpRight } from "#/assets/icons"
+
+const NAVIGATION: Pick<LinkProps, "href" | "isExternal" | "children">[] = [
```

Comment:
I think we can pick just `href` and `children` and set the `isExternal` to `true` in Footer component.

Reply:
Ref commit: ab4881a4812cc7cdb4f2da954b3ae546ed6310e8